test/integration/test_build.py

```diff
@@ -135,6 +135,16 @@ def test_build_args_basic(cli, runtime, ee_tag, tmp_path, data_dir):
     assert 'FOO=bar' in result.stdout
 
 
+@pytest.mark.test_all_runtimes
```

Comment:
We can probably cover this adequately with decent unit tests on either the CLI parser or maybe even just the `BuildArgsAction` itself- another dedicated integration test is incredibly expensive for this case.